kytos/core/interface.py

```diff
@@ -318,9 +362,12 @@ def use_tags(
         Exceptions:
             KytosTagsAreNotAvailable from _use_tags()
         """
+        if tag_type != TAGType.VLAN.value:
```

Comment:
At the moment, this won't lead to issues, but it's hardcoding something that it's currently configurable, it should be checking if tag_type (the key) is on `self.available_tags` or `self.special_available_tags` accordingly, or more easily just catching `KeyError`. Having a `tag_type != TAGType.VLAN.value` on methods that are setting the tag ranges as you've done on `set_tag_ranges` it's OK, but now on methods that are using or making the tags available that shouldn't be the case, since they should only rely on whatever dict keys as types are available. See what I mean?

Reply:
Got it. If `tags` is ordered, `_make_tags_available()` and `_used_tags()` can only raise KeyError when `tag_type` is not supported so these methods are inside a `try except` clause.